### DISCUSSION TITLE
RabbitMQ version

### DISCUSSION THREAD

**Discussion by Trapulo** [postalserver/postal#2047] [Help with using Postal]
Hi,
I noticed that the guide installs RabbitMQ from the v. 3.8 repository. Is this mandatory, or I can use the latest image?

thanks

**Comment by willpower232** [comment]
Presuming they're following semver, you should be good with the latest 3.whatever version

**Reply by Trapulo** [reply]
I installed with the "latest" tag and seems fine. Maybe the doc can be changed... ?

**Reply by willpower232** [reply]
Good to know, I probably would have gone with `3` instead of `latest` in case they release a version 4 one day which is completely different.
